ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLandOwnership.vue

```diff
@@ -1,49 +1,135 @@
 <template>
-  <v-card flat rounded id="mhr-home-land-ownership" class="mhr-home-land-ownership pa-8">
-    <v-row no-gutters>
-      <v-col cols="12" sm="3">
-        <label class="generic-label" for="ownership">
-          Land Lease or Ownership
-        </label>
-      </v-col>
-      <v-col cols="12" sm="9">
-        <v-checkbox
-          id="ownership"
-          label="The manufactured home is located on land that the homeowners own,
-                 or on which they have a registered lease of 3 years or more."
-          v-model="isOwnLand"
-          class="my-0 py-0 px-0 ownership-checkbox"
-          data-test-id="ownership-checkbox"
-        />
-       </v-col>
-    </v-row>
+  <v-card flat rounded id="mhr-home-land-ownership"
+          class="mhr-home-land-ownership pa-8">
+    <v-form ref="leaseOrOwnForm">
+      <v-row no-gutters>
+        <v-col cols="12" sm="3">
+          <label class="generic-label" :class="{'error-text': validate}">
+            Land Lease or Ownership
+          </label>
+        </v-col>
+        <v-col cols="12" sm="9">
+          <p>
+            Is the manufactured home located on land that the homeowners own or on land that
+            they have a registered lease of 3 years or more?
+          </p>
+        </v-col>
+        <v-row class="mt-0 mb-n5">
+          <v-col cols="10" offset="3">
+            <v-radio-group
+              id="lease-own-option"
+              v-model="isOwnLand"
+              class="mt-2 ml-n2 mb-3"
+              row
+              data-test-id="ownership-radios"
+            >
+              <v-radio
+                id="yes-option"
+                class="yes-radio"
+                label="Yes"
+                active-class="selected-radio"
+                :value="true"
+                data-test-id="yes-ownership-radiobtn"
+              />
+              <v-radio
+                id="no-option"
+                class="no-radio"
+                label="No"
+                active-class="selected-radio"
+                :value="false"
+                data-test-id="no-ownership-radiobtn"
+              />
+            </v-radio-group>
+          </v-col>
+        </v-row>
+        <v-row v-if="isOwnLand">
+          <v-col cols="9" offset="3">
+            <v-divider class="mx-0 divider-mt" />
+            <p class="mb-n2 paragraph-mt" data-test-id="yes-paragraph">
+              <b>Note:</b> Land ownership or registered lease of the land for 3 years or more
+              must be verifiable through the BC Land Title and Survey Authority (LTSA)
+              or other authorized land authority.
+            </p>
+          </v-col>
+        </v-row>
+        <v-row v-if="!isOwnLand && isOwnLand!=null">
```

Comment:
Could we use `v-else` here, if this is the other condition for the previous `v-if`?

Reply:
Nothing should be rendered if isOwnLand === null, so v-else should not be used here!